README.md

```diff
@@ -18,14 +18,34 @@ and outlines an API that we think solves this problem. But this is less about
 proposing an API, and more about starting a conversation, so take a look, and
 let us know what you think (contact details below).
 
+## Demo
+
+We have a prototype that works in Chrome and Firefox. Try this:
+
+1. Go to
+   [handler-dot-chromium-ballista.appspot.com](https://handler-dot-chromium-ballista.appspot.com)
+   (Ballista Editor Demo), and register as an action handler.
```

Comment:
This reads strangely. The person isn't registering as an action handler, they're allowing the app to register.